jokes/index.json

```diff
@@ -199,8 +199,22 @@
   },
   {
     "id": 33,
+    "type": "knock-knock",
+    "setup": "Knock knock. \n Who's there? \n Opportunity.",
+    "punchline": "That is impossible. Opportunity doesn�t come knocking twice!"
+  },
+  {
+    "id": 34,
+    "type": "programming",
+    "setup": "Why is it that women find C to be more attractive than Java?",
+    "punchline": "Because C doesn�t treat them like objects."
+  },
+  {
+    "id": 34,
```

Comment:
Make sure each id is unique as well. You have two 34's. This will be fine though once the joke about women is removed.